lxd/patches.go

```diff
@@ -34,6 +35,7 @@ var patches = []patch{
 	{name: "leftover_profile_config", run: patchLeftoverProfileConfig},
 	{name: "network_permissions", run: patchNetworkPermissions},
 	{name: "storage_api", run: patchStorageApi},
+	{name: "storage_api", run: patchStorageApi},
```

Comment:
The name must be different, the "run:" function can remain the same.

So change it to:

    {name: "storage_api_1", run: patchStorageApi},